sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -98,10 +99,10 @@ async def get_chat_completions_client(self, **kwargs) -> "ChatCompletionsClient"
 
         try:
             from azure.ai.inference.aio import ChatCompletionsClient
-        except ModuleNotFoundError as _:
+        except ModuleNotFoundError as e:
```

Comment:
Isn't this going to cause another quality gate error, saying "e" is not being used?

Reply:
No, `e` is used on line 105

https://github.com/Azure/azure-sdk-for-python/blob/f7c959b47eb62eb1f7a2b847f3384bae8f9648b5/sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py#L105